tools/internal/ios/core-test-runner/core-test-runner.xcodeproj/project.pbxproj

```diff
@@ -548,7 +548,7 @@
 			repositoryURL = "https://github.com/iTwin/mobile-native-ios";
 			requirement = {
 				kind = exactVersion;
-				version = 3.3.12;
+				version = 3.3.13;
```

Comment:
just to confirm, there is no pbxproj in dta/ios folder yet right?

Reply:
There's a pbxproj but no XCRemoteSwiftPackageReference to update.

Reply:
ok cool; guess still waiting on my bp; ill check if its working properly on 3.3.x tomorrow after this is merged in